src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -721,5 +724,10 @@ private void discoverModCompat()
         {
             Compatibility.beeHiveCompat = new ResourcefulBeesCompat();
         }
+        if (ModList.get().isLoaded("tconstruct"))
+        {
+            Compatibility.tinkersCompat = new TinkersWeaponHelper();
+            Compatibility.tinkersSlimeCompat = new SlimeTreeCheck();
```

Comment:
you probably could just do the same with the toolbroken here

Reply:
That one doesn't have a super class yet though. But I could either create one or just relocate the method to the WeaponProxy/Helper

Reply:
ye I guess you could just make it a toolHelper, its similar enoguh

Reply:
like rename the weaponhelper to toolhelper and throw it in there